docs/notebooks/regional_frequency_analysis.ipynb

```diff
@@ -0,0 +1,970 @@
+{
```

Comment:
<p><code>calculate the values</code> est flou. "calculate the return periods"?</p><p> </p><p>Aussi, il manque d'info sur ce que <code>remove_small_regions</code>  fait exactement.</p><ul><li>Mentionner rapidement que le défaut est de 5 stations.</li><li>Qu'est-ce qui se passe pour les clusters avec moins de 5 régions ?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>